### PR TITLE
Overhauled info

### PR DESCRIPTION
**Pull request by ben9583**
Ok so a lot of stuff has happened:

-Removed fontawesome link cause it took extra time to load (also that wasn't our code...)
-Created an info button
-Animations and color changed based on which tab is opened
-Fixed a bug when toggling fetch would not transition background color (this took the most time, honestly. Turns out, it was because I thought 93 + 8 == 100.)
-When you click the info button, a tab drops down from above. When you click it again, it flys up away

Basically, I mostly just worked to replace that fontawesome template and make our OWN button. We shall not be dependant on the technocrat, bourgeois fontawesome developers. We will strive on autarky of the people!

### CODE REVIEW

**Comment by ben9583** [comment]
oh yea btw I changed the background color to neon red and the info-pane is neon green for debug purposes. The red should never show and I'll let the next project work on the info-pane.
